### ISSUE TITLE
SC3046 false positive in multistage Dockerfile

### ISSUE THREAD

**Issue by powerman** [hadolint/hadolint#1174]
```Dockerfile
FROM ubuntu:24.04 AS stage
SHELL ["/bin/bash", "-e", "-o", "pipefail", "-c"]

FROM stage AS test
RUN source /etc/os-release
```

```console
$ hadolint -v
Haskell Dockerfile Linter 2.14.0
$ hadolint Dockerfile
Dockerfile:5 SC3046 warning: In POSIX sh, 'source' in place of '.' is undefined.
```

**Comment by gremo** [comment]
Same problem here, any update?

**Comment by m-ildefons** [comment]
Fixed in v2.15.0
